include/irr/asset/IAssetWriter.h

```diff
@@ -20,17 +20,10 @@ namespace irr { namespace asset
 */
 enum E_WRITER_FLAGS : uint32_t
```

Comment:
its important to note that not every writer supports all these flags and may just silently drop them when writing (example, JPG, PNG do not write encrypted images)

Reply:
status?